pages/analysis.md

```diff
@@ -1,15 +1,132 @@
+{% include nav.html %} {% include css.html %}
+
 # Analysis
+---
+	
+The visualization on this page represents the documented interference
+on the slave trade by privateers and pirates between 1699-1827. The primary 
+geographical focus is in the Caribbean and West Indies region. These regions 
+are where the most slave trade activity took place. The original hypothesis 
+for this research is that privates and privateers had a significant effect on 
+the slave trade. Pirates and privateers caused fiscal strains on the slave t
+rade market and economically effected countries that participated in the slave 
+trade. As per the documented information, the slave trade was extremely expansive
+that the incursion by privateers had little to no effect on the market itself. 
+
+The images do raise questions to the effects privateers and pirates had on smaller
+countries. These smaller states would not have had such availability to the slave 
+trade if not provided by the captive ships. The images below demonstrate the 
+disparities and theory.
+
+---
+
+![image]({{ site.baseurl}}/assets/images/1.%20Origin%20Ports.png)
```

Comment:
These are great visuals, the only hindrance is that you can't see them full-size. If you want to get more out of it, you could make the picture linked (clickable), so that the reader can see the literal big picture. To do that you just put the entire code you have for the picture inside the square brackets of a link, and then you recopy the URL of the picture inside the parentheses of a link, like this [the entire code you have on line 23 goes here](the link to the picture is repeated here). I have more full comments and an example on the workflow page if you want more info.